### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by kidde60**
### **Validation Form**
**On this webpage I:**

- added a validation form with JavaScript.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -99,144 +99,144 @@ cardArr.forEach((item, i) => {
...
+  // button.name = 'details';
+  // button.type = 'button';
+  // button.id = cardArr[i].id;
+  // stories.appendChild(button);
+  // // Card Button function
+  // function openPopup(e) {
+  //   const j = e.target.id;
+  //   const popup = document.createElement('div');
+  //   popup.className = 'popup';
+  //   workSection.appendChild(popup);
```
Comment (chrystalme) [inline script.js]: Kindly remove commented out code from your code base.

File: script.js [hunk 2]
```diff
@@ -99,144 +99,144 @@ cardArr.forEach((item, i) => {
...
 
-    function closePopup() {
-      popup.style.display = 'none';
-    }
+  //   function closePopup() {
+  //     popup.style.display = 'none';
+  //   }
```
Comment (chrystalme) [inline script.js]: Kindly remove commented out code from your code base.

File: script.js [hunk 3]
```diff
@@ -99,144 +99,144 @@ cardArr.forEach((item, i) => {
...
 
-    const popupContent = document.createElement('div');
-    popupContent.className = 'popupContent fontSmall';
-    popup.appendChild(popupContent);
+  //   const popupContent = document.createElement('div');
+  //   popupContent.className = 'popupContent fontSmall';
+  //   popup.appendChild(popupContent);
```
Comment (chrystalme) [inline script.js]: Kindly remove commented out code from your code base.

File: script.js [hunk 4]
```diff
@@ -99,144 +99,144 @@ cardArr.forEach((item, i) => {
...
-    img = document.createElement('img');
-    img.src = 'images/Icon.svg';
-    img.className = 'close';
-    popupContent.appendChild(img);
-    img.addEventListener('click', closePopup);
+  //   img = document.createElement('img');
+  //   img.src = 'images/Icon.svg';
+  //   img.className = 'close';
+  //   popupContent.appendChild(img);
+  //   img.addEventListener('click', closePopup);
```
Comment (chrystalme) [inline script.js]: Kindly remove commented out code from your code base.

File: script.js [hunk 5]
```diff
@@ -99,144 +99,144 @@ cardArr.forEach((item, i) => {
...
 
-    const popupHeader = document.createElement('h2');
-    popupHeader.textContent = cardArr[j].name;
-    popupContent.appendChild(popupHeader);
+  //   const popupHeader = document.createElement('h2');
+  //   popupHeader.textContent = cardArr[j].name;
+  //   popupContent.appendChild(popupHeader);
```
Comment (chrystalme) [inline script.js]: Kindly remove commented out code from your code base.

**Review by chrystalme — CHANGES_REQUESTED** [review]
# Hi @kidde60  and  @ababaug, 

You have done a good job so far!
However, there are some issues that you still need to work on to prepare your project for further evaluation

## Highlights 💡 
- Linter properly configured ✅ 

## Suggested changes ♻️ 
_Check the comments under the review._
 - Also, your error is not being displayed above the submit button.
 - Kindly check your code and submit another pull request when your code is working.

## Optional Changes
 - Kindly add your partner's information to your readme file.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA). Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi TEAM, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**